### PR TITLE
Provide default logger group explicitly

### PR DESCRIPTION
**Pull request by xDimon**
Deprecate star-marked default group. PR provides explicitly defined default group (new soralog feature).

Signed-off-by: Dmitriy Khaustov aka xDimon <khaustov.dm@gmail.com>

### CODE REVIEW

File: example/02-kademlia/rendezvous_chat.cpp [hunk 1]
```diff
@@ -175,7 +179,11 @@ namespace {
 int main(int argc, char *argv[]) {
   // prepare log system
   auto logging_system = std::make_shared<soralog::LoggingSystem>(
-      std::make_shared<libp2p::log::Configurator>(logger_config));
+      std::make_shared<soralog::ConfiguratorFromYAML>(
```
Comment (igor-egorov) [inline example/02-kademlia/rendezvous_chat.cpp:182]: Logger initialization looks a bit wordy now.

**Review by igor-egorov — APPROVED** [review]

**Review by art-gor — APPROVED** [review]

File: example/02-kademlia/rendezvous_chat.cpp (hunk 1 above)
Comment (xDimon) [reply example/02-kademlia/rendezvous_chat.cpp:182]: No, it's correct. It is needed to apply libp2p's part of config before app's part
